lib/configure.js

```diff
@@ -137,6 +139,18 @@ function configure (gyp, argv, callback) {
     // disable -T "thin" static archives by default
     variables.standalone_static_library = gyp.opts.thin ? 0 : 1
 
+    if (win && !(gyp.opts.msvs_version && gyp.opts.msvs_version !== '2017')) {
+      const vsSetup = findVS2017()
+      if (vsSetup) {
+        gyp.opts.msvs_version = '2015'
```

Comment:
are we forcing this to make GYP happy, or is this (and the next line) a mistake?